resources/views/fixometer/device-add-or-edit.blade.php

```diff
@@ -0,0 +1,231 @@
+                                        <option value="{{ $device->brand }}" selected>{{ $device->brand }}</option>
+                                    @endif
+                                </select>
+                            </div>
+                            <div></div>
+                        </div>
+
+                        <div class="mb-2 device-select-row">
+                            <div class="form-group">
+                                <input type="text" class="form-control field" name="model" value="{{ $device->model }}" placeholder="@lang('partials.model')" autocomplete="off">
+                            </div>
+                            <div data-toggle="popover" data-placement="left" data-html="true" data-content="@lang('devices.tooltip_model')" class="ml-3 mt-2">
+                                <img src="/icons/info_ico_black.svg">
+                            </div>
+                        </div>
+
+                        <div class="device-field-row align-items-center mb-2">
+                            <label class="text-white text-bold">
+                                @lang('devices.weight')*
+                            </label>
+                            <div class="display-weight">
+                                <div class="input-group">
+                                    <input disabled type="number" class="weight form-control form-control-lg field numeric" name="weight" min="0.01" step=".01" autocomplete="off" value="{{ $device->estimate }}">
+                                </div>
+                            </div>
+                            <span class="text-white text-right mb-1">
+                            @lang('devices.required_impact')
+                        </span>
+                        </div>
+
+                        <div class="device-field-row align-items-center mb-2">
+                            <label class="text-black text-bold">
+                                @lang('devices.age')
+                            </label>
+                            <div class="display-weight">
+                                <div class="input-group">
+                                    <input type="number" class="form-control field" name="age" min="0" step="0.5" value="{{ $device->age }}" autocomplete="off">
+                                </div>
+                            </div>
+                            <span class="text-black text-right mb-1">
+                            @lang('devices.age_approx')
+                        </span>
+                        </div>
+{{--                        <p class="text-danger">TODO Add photos before add device, but only if quantity is 1.</p>--}}
+                    </div>
+                </div>
+            </div>
+
+            <div class="card card-event-add-item flex-grow-1 border border-top-0 border-bottom-1 border-left-0 border-right border-white">
+                <div class="card-body">
+                    <h3>REPAIR</h3>
```

Comment:
Will need wrapping in @lang

Reply:
Do you want me to do that as part of this?  If you search for e.g. H3, there are very many headers which are not translated.  I think that's worth its own story.

Reply:
Yes please - it's a new addition to our definition of done, that any new text we add must be translatable, and we alert our translators to it when the new feature is made live. Re: old text, that is something we have been working through with Repair Together, see e.g. https://github.com/TheRestartProject/restarters.net/commit/438b4401e09d025d435dde2ad42469d5b5f3e934

Reply:
Fair enough.  I've pushed some changes with translations in view and device-add-or-edit - let me know if you spot any others.